app/controllers/concerns/bbb_server.rb

```diff
@@ -54,6 +54,7 @@ def join_path(room, name, options = {}, uid = nil)
     join_opts = {}
     join_opts[:userID] = uid if uid
     join_opts[:join_via_html5] = true
+    join_opts[:guest] = true if options[:require_moderator_approval] && !options[:user_is_moderator]
```

Comment:
I have no idea if it's proper to post my comment here, but this fix gets the guest-wait page back to life. Thanks.